emcc.py

```diff
@@ -394,18 +394,32 @@ def find_output_arg(args):
   """
   outargs = []
   specified_target = None
-  use_next = False
-  for arg in args:
-    if use_next:
-      specified_target = arg
-      use_next = False
-      continue
+
+  arg_count = len(args)
+  arg_index = 0
+
+  while arg_index < arg_count:
+    arg = args[arg_index]
+
     if arg == '-o':
-      use_next = True
-    elif arg.startswith('-o'):
+      if arg_index != arg_count - 1:
+        specified_target = args[arg_index + 1]
+      arg_index += 2
+      continue
+
+    if arg.startswith('-o'):
       specified_target = arg[2:]
+      arg_index += 1
+      continue
+
+    outargs.append(arg)
+
+    if arg == '-mllvm' and arg_index != arg_count - 1:
```

Comment:
How about a commit here.   Maybe something like `Explicitly skip over `-mllvm` arguments because they could potentially start with `-o` and be confused for output file specifiers`?